packages/taquito/src/constants.ts

```diff
@@ -41,7 +42,8 @@ export const protocols = {
   '010': [Protocols.PtGRANADs],
   '011': [Protocols.PtHangz2], // hangzhou v2,
   '012': [Protocols.PsiThaCa], //Ithaca
-  '013': [Protocols.ProtoALpha],
+  '013': [Protocols.Psithaca2], //Ithaca2
+  '014': [Protocols.ProtoALpha],
 };
 
 export enum DefaultLambdaAddresses {
```

Comment:
A lambda contract and the chain_id for ithaca2 are missing in this file.

Reply:
originated the Lambda contract and added the address here
added the chon_id